journal/modernized/03-chapter-2.adoc

```diff
@@ -455,6 +472,8 @@ Pray you for us, and for me, in an especial manner.
 [.signed-section-signature]
 Ann Branson
 
+--
+
```

Comment:
I'm hoping I did this right, I only worked on one chapter until you guys have a chance to look at it. Thanks!!